mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -181,7 +188,8 @@ def get_datetimes(ds, timestr, yearoffset):  # {{{
 
 
 def preprocess_mpas(ds, onlyvars=None, selvals=None, iselvals=None,
-                    timestr='xtime', yearoffset=1849):  # {{{
+                    timestr='xtime', yearoffset=1849,
+                    variable_map=None):  # {{{
```

Comment:
Oh, man.  I *really* hate these all lowercase variable names.  Even an underscore is a *big* compromise for me.  I super-duper prefer mixed case.  But this is your library so I'll do what you like.